observer/observer.go

```diff
@@ -67,7 +64,14 @@ func (o *Observer) processBlock(events chan<- Event, block *blockatlas.Block) {
 					Decimals: coin.Coins[o.Coin].Decimals,
 				}
 			}
-			emitted[tx.ID] = true
+			if d, ok := emitted[tx.ID]; ok {
+				if d == tx.Direction || d == blockatlas.DirectionSelf{
```

Comment:
What’s different with just true?

Reply:
because tx going to send just one time per observer, now we filter by direction, the tx going to sent one time per direction and per observer

Reply:
It’s for utxo only?

Reply:
yes

Reply:
if for all, account based transactions does not have direction type, it's empty I assume it's just for utxo then?

Reply:
sorry, I thought all type of tx we infer the direction. I am going to change this